.gitignore

```diff
@@ -1,104 +1,23 @@
-# Logs
-logs
```

Comment:
Project looks awesome! README file should have a project name, description about project features, and instructions for users to set it up locally. Moreover, please remove unnecessary details like `Author2`. Kindly remember that README is the first point of interaction for your project so it should be clean and professional!